docs/building-a-form/available-components.md

```diff
@@ -348,18 +416,27 @@ When you build a form with more than one chapter (shown by the segments in a pro
 
 #### Usage guidelines
 
-Break into separate topic?
+In general, the review page renders the form data in review mode automatically. There are, however, some specific options you can pass to the form config to customize certain review functionality.
```

Comment:
> In general, the review page renders the form data in review mode automatically. There are, however, some specific options you can pass to the form config to customize certain review functionality.

How about:

"The review page renders the form data in review mode automatically. However, you can pass some specific options to the form config to customize some review features."

Reply:
@annekainicUSDS, when we discussed "Review page" last week as a group here, we thought it might be a good idea to break the information into a separate topic. Given this information you've provided here (which doesn't seem overly complicated), do you think we're ok keeping it here with the other form features?

Reply:
I think it's okay to keep it in this file, as there isn't that much to know about review pages.